runtimes/standalone/Cargo.toml

```diff
@@ -216,6 +216,11 @@ git = "https://github.com/paritytech/substrate"
 package = "pallet-timestamp"
 version = "3.0.0"
 
+[dependencies.pallet-utility]
```

Comment:
Why do we have mixed dependency styles here? Should we make it consistent?

Reply:
this toml uses the other stile. So in the toml it's not mixed. 😅

Reply:
I see a `[dependencies]` section tho...

Reply:
Ah yes. didn't see that. So IMHO we should use the short version for simple dependencies and the expended version for dependencies that use multiple options.

Reply:
on the other hand it becomes quite verbose and long. So I'm undecided.

Reply:
In my opinion it would be more chaotic, and also I think we have been converging to using the short version in the other pallets. So we should probably use it everywhere, that's my 2 cents here.